pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -364,22 +313,15 @@ func (r *ReconcileChaosEngine) getChaosEngineInstance(engine *chaosTypes.EngineI
 
 // Get application details
 func getApplicationDetail(engine *chaosTypes.EngineInfo) error {
-	applicationInfo := &chaosTypes.ApplicationInfo{}
-	appInfo, err := initializeApplicationInfo(engine.Instance, applicationInfo)
-	if err != nil {
-		return err
-	}
-	engine.AppInfo = appInfo
 
 	var appExperiments []string
-	for _, exp := range appInfo.ExperimentList {
+	for _, exp := range engine.Instance.Spec.Experiments {
 		appExperiments = append(appExperiments, exp.Name)
 	}
 	engine.AppExperiments = appExperiments
 
-	chaosTypes.Log.Info("App key derived from chaosengine is ", "appLabelKey", chaosTypes.AppLabelKey)
-	chaosTypes.Log.Info("App Label derived from Chaosengine is ", "appLabelValue", chaosTypes.AppLabelValue)
-	chaosTypes.Log.Info("App NS derived from Chaosengine is ", "appNamespace", appInfo.Namespace)
+	chaosTypes.Log.Info("App Label derived from Chaosengine is ", "appLabel", engine.Instance.Spec.Appinfo.Applabel)
```

Comment:
Lets remove `is`. In structured logging, these would come up as key value pair. So, we don't need it.
```suggestion
	chaosTypes.Log.Info("App Label derived from Chaosengine","chaosEngineName",engine.Instance.Name,"chaosEngineNamespace", engine.Instance.Namespace, "appLabel", engine.Instance.Spec.Appinfo.Applabel)
```